capability/capability_test.go

```diff
@@ -151,3 +151,40 @@ func TestAmbientCapSet(t *testing.T) {
 		}
 	}
 }
+
+func TestApplyCapsForOtherProcess(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		return
+	}
+	requirePCapSet(t)
+
+	cmd := exec.Command("sleep", "sleep", "infinity")
```

Comment:
I think this should be 
```suggestion
	cmd := exec.Command("sleep", "infinity")
```

Reply:
The error went unnoticed because this test is not being run, because the previous test (TestAmbientCapSet) removes CAP_SET_PCAP from the test process. Oh well

Reply:
See [here](https://github.com/moby/sys/pull/171#pullrequestreview-2376989867)